src/display.c

```diff
@@ -97,11 +98,37 @@ static int display_start_dcp(void)
     return 0;
 }
 
-int display_parse_mode(const char *config, dcp_timing_mode_t *mode)
+int display_parse_mode(const char *config, dcp_timing_mode_t *mode, int *wait_delay)
 {
     memset(mode, 0, sizeof(*mode));
+    *wait_delay = 0;
 
-    if (!config || !strcmp(config, "auto"))
+    if (!config)
```

Comment:
I would keep the mode parsing code as is. I think it makes sense to keep it as first parameters for the `display=` variable. Have a look how I added support for comma separated parameters in [#212](https://github.com/AsahiLinux/m1n1/pull/212/commits/07e54b52e6cf228fca02f973728722d241ae09c1)